test

```diff
@@ -362,7 +362,7 @@ function shellcheck_pass {
 
 function markdown_you_pass {
 	# eschew you
-	yous=$(find . -name \*.md -exec grep -E --color "[Yy]ou[r]?[ '.,;]" {} + | grep -v /v2/ || true)
+	yous=$(find . -name \*.md ! -path './vendor/*' ! -path './Documentation/v2/*' -exec grep -E --color "[Yy]ou[r]?[ '.,;]" {} + || true)
```

Comment:
@gyuho We're okay with excluding `/vendor`, right?  I've also switched to `! -path` style filtering here since it's already used elsewhere in this test file and it make the file more consistent.

Reply:
@jpbetz Yes. I tried your command, and confirm that it fixes. We want to exclude `vendor`. Seems like it's been broken since :0